.spin/cmds.py

```diff
@@ -99,3 +99,18 @@ def ipython(ctx, ipython_args):
     ) + ipython_args
 
     ctx.forward(meson.ipython)
+
+
+@click.command()
+def sdist():
+    """📦 Build a source distribution in `dist/`"""
+    p = util.run(["pyproject-build", ".", "--sdist"], output=False)
+    built_line = next(
+        line
+        for line in p.stdout.decode('utf-8').split('\n')
+        if line.startswith('Successfully built')
+    )
+    print(built_line)
```

Comment:
```suggestion
    output = p.stdout.decode("utf-8")
    if "Successfully built" in output:
        print(built_line)
    else:
        print(output)
```
Seems a bit clearer.

Reply:
The run command prints output upon failure.